src/rt/util/hash.d

```diff
@@ -95,15 +90,18 @@ size_t hashOf( const(void)* buf, size_t len, size_t seed )
     return hash;
 }
 
-// Check that hashOf works with CTFE
-@nogc nothrow pure unittest
```

Comment:
The printf is used to see what the hash value is so it can be put in the assert.
Attributes for unittests make sense when they are verifying correct attribute inference for templates. It is pointless here.